c2_components/include/mfx_c2_decoder_component.h

```diff
@@ -240,5 +240,5 @@ class MfxC2DecoderComponent : public MfxC2Component
     std::list<std::shared_ptr<mfxFrameSurface1>> m_surfacePool; // used in case of system memory
 
     unsigned int m_uOutputDelay = 8u;
-    unsigned int m_uInputDelay = 0u;
+    unsigned int m_uInputDelay = 12u;
```

Comment:
Why set it to 12, but not others value, can you explain ?

Reply:
Same concern. @zhangyichix , does this case pass with google c2 sw decoder? If so, please help check the input buffer count which is used by hevc c2 sw decoder.